src/datetimepicker.story.jsx

```diff
@@ -119,4 +119,21 @@ storiesOf('DateTimePicker', module)
 				datepickerOptions={opts}
 			/>
 		</div>);
+	})
+	.add('error state', () => {
+		const opts = {
+				allowInput: true,
+				minDate: Date.now()
+			},
+			date = new Date(3000,1,1,15,0,0);
+		return (<div className='span--50'>
+			<DateTimePicker
+				name='event_time'
+				label='Start at'
+				value={date}
+				error={'Whoops, something went wrong.'}
+				datepickerOptions={opts}
+				forceCalendar
+			/>
+		</div>);
```

Comment:
doesn't matter much because it's a storybook story, but this would be easier to read if the JSX was fully nested inside the parens

```jsx
return (
  <div ...
  </div>
);
```